sdk/spring/spring-cloud-azure-starter-integration-eventhubs/pom.xml

```diff
@@ -101,4 +101,24 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.1.2</version> <!-- {x-version-update;org.apache.maven.plugins:maven-jar-plugin;external_dependency} -->
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>com.azure.spring.integration.eventhubs</Automatic-Module-Name>
```

Comment:
This is not correct, this module is a starter. should use this one `com.azure.spring.cloud.starter.integration.eventhubs`